### PR TITLE
bind wlr_keyboard_shortcuts_inhibit_v1.h

### PR DESCRIPTION
**Pull request by novakne**

### CODE REVIEW

**Comment by novakne** [comment]
Fix issues found by the test, should have tried this before pushing the rebase

File: src/types/keyboard_shortcuts_inhibit_v1.zig [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const wlr = @import("../wlroots.zig");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+pub const KeyboardShortcutsInhibitManagerV1 = extern struct {
+    inhibitors: wl.list.Head(wlr.IdleInhibitorV1, "link"),
```
Comment (ifreund) [inline src/types/keyboard_shortcuts_inhibit_v1.zig:7]:
```suggestion
    inhibitors: wl.list.Head(wlr.KeyboardShortcutsInhibitorV1, "link"),
```

File: src/types/keyboard_shortcuts_inhibit_v1.zig [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    global: ?*wl.Global,
```
Comment (ifreund) [inline src/types/keyboard_shortcuts_inhibit_v1.zig:8]:
```suggestion
    global: *wl.Global,
```

**Review by ifreund — CHANGES_REQUESTED** [review]

**Review by ifreund — APPROVED** [review]
LGTM, thanks!

**Pull request merged**